x-pack/plugins/apm/public/services/rest/apm/transaction_groups.ts

```diff
@@ -72,31 +85,23 @@ export async function loadDetailsCharts({
   });
 }
 
-export async function loadOverviewCharts({
+export async function loadTransactionOverviewCharts({
   serviceName,
   start,
   end,
   kuery,
-  transactionType = 'request'
+  transactionType
 }: IUrlParams) {
-  return callApi<TimeSeriesAPIResponse>({
-    pathname: `/api/apm/services/${serviceName}/transaction_groups/${transactionType}/charts`,
-    query: {
-      start,
-      end,
-      esFilterQuery: await getEncodedEsQuery(kuery)
-    }
-  });
-}
+  if (!(serviceName && start && end)) {
+    throw new MissingArgumentsError();
+  }
+
+  const pathname = transactionType
+    ? `/api/apm/services/${serviceName}/transaction_groups/${transactionType}/charts`
```

Comment:
good catch. It's the same fix i used for service detail tabs (https://github.com/elastic/kibana/pull/33828/files#diff-454d6093ad5041c60ad3541cd915bd5aR33)

Reply:
Ah, fun :)
I opted for fixing it here, since it allowed me to delete `loadOverviewChartsForAllTypes`.